src/api/v1/community_businesses/volunteer_logs.ts

```diff
@@ -431,14 +431,7 @@ const routes: Hapi.ServerRoute[] = [
         {
           fields: ['duration'],
           where: { organisationId: communityBusiness.id },
-          whereBetween: since || until
-            ? {
-              startedAt: [
-                since || new Date(0),
-                until || new Date(),
-              ],
-            }
-            : undefined,
+          whereBetween: { startedAt: [since, until] },
```

Comment:
how come the default `new Date` has been removed?

Reply:
Because that branch was unreachable. `since` and `until` are both already `Date` objects (see line 426), so the `||` condition will never fall through.